{"name": "test_delete_pet_by_id", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "api_client = <API.client.api_client.ApiClient object at 0x000002AC75836E50>\n\n    def test_delete_pet_by_id(api_client):\n        response1 = api_client.pet.delete_pet_by_id(3)\n>       assert response1.status_code == 200\nE       assert 404 == 200\nE        +  where 404 = <Response [404]>.status_code\n\nAPI\\tests_api\\test_poc.py:29: AssertionError"}, "start": 1727471787051, "stop": 1727471787578, "uuid": "08084384-2fd3-4d75-a280-644489567a72", "historyId": "8035f3d5d02456803cddfb69c8d472f9", "testCaseId": "8035f3d5d02456803cddfb69c8d472f9", "fullName": "API.tests_api.test_poc#test_delete_pet_by_id", "labels": [{"name": "parentSuite", "value": "API.tests_api"}, {"name": "suite", "value": "test_poc"}, {"name": "host", "value": "viktoria_hard"}, {"name": "thread", "value": "12180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "API.tests_api.test_poc"}]}